### DISCUSSION TITLE
Ranger Image Previewing on macOS

### DISCUSSION THREAD

**Discussion by jcguu95** [ghostty-org/ghostty#3978] [Q&A] [answered]
Has anyone had ghostty and ranger's image preview working together on macOS? 

Not being able to preview images is the only reason I'm still with `kitty`. I know `ghostty` has Kitty image protocol, but I couldn't make it work within ranger.

A related discussion on linux: https://github.com/ghostty-org/ghostty/issues/2055

**Comment by edrozenberg** [comment]
No luck here either with ranger. I do get image previews just fine with viu (viewer), chafa (viewer), and yazi (file mgr).

**Comment by 00-kat** [comment, marked as answer]
Related: https://github.com/ranger/ranger/pull/3036.
